src/components/claims/claims-list/components/ClaimListHeader.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Table } from 'semantic-ui-react'
+
+type SortDirection = 'ascending' | 'descending'
```

Comment:
We could create an enum here instead:
```suggestion
enum SortDirection {
  Ascending = 'ASC',
  Descending = 'DESC'
}
```